tests/integration/ie-tests.sh

```diff
@@ -76,20 +88,20 @@ test_pyspark() {
   echo "sc = SparkContext()" >> test-pyspark.py
   echo "spark = SparkSession(sc)" >> test-pyspark.py
   echo "for conf in spark.sparkContext.getConf().getAll(): print (conf)" >> test-pyspark.py
-  echo "partitions = 1000" >> test-pyspark.py
-  echo "n = 100000 * partitions" >> test-pyspark.py
+  echo "partitions = 10" >> test-pyspark.py
```

Comment:
i just realised that these tests are written in shell not python. They need to be in python and using a standard python unit/integration testing library.